tests/tests/cpld_s2sw_wavice_p8

```diff
@@ -0,0 +1,95 @@
+                   RESTART/20210323.060000.fv_core.res.tile3.nc \
+                   RESTART/20210323.060000.fv_core.res.tile4.nc \
+                   RESTART/20210323.060000.fv_core.res.tile5.nc \
+                   RESTART/20210323.060000.fv_core.res.tile6.nc \
+                   RESTART/20210323.060000.fv_srf_wnd.res.tile1.nc \
+                   RESTART/20210323.060000.fv_srf_wnd.res.tile2.nc \
+                   RESTART/20210323.060000.fv_srf_wnd.res.tile3.nc \
+                   RESTART/20210323.060000.fv_srf_wnd.res.tile4.nc \
+                   RESTART/20210323.060000.fv_srf_wnd.res.tile5.nc \
+                   RESTART/20210323.060000.fv_srf_wnd.res.tile6.nc \
+                   RESTART/20210323.060000.fv_tracer.res.tile1.nc \
+                   RESTART/20210323.060000.fv_tracer.res.tile2.nc \
+                   RESTART/20210323.060000.fv_tracer.res.tile3.nc \
+                   RESTART/20210323.060000.fv_tracer.res.tile4.nc \
+                   RESTART/20210323.060000.fv_tracer.res.tile5.nc \
+                   RESTART/20210323.060000.fv_tracer.res.tile6.nc \
+                   RESTART/20210323.060000.phy_data.tile1.nc \
+                   RESTART/20210323.060000.phy_data.tile2.nc \
+                   RESTART/20210323.060000.phy_data.tile3.nc \
+                   RESTART/20210323.060000.phy_data.tile4.nc \
+                   RESTART/20210323.060000.phy_data.tile5.nc \
+                   RESTART/20210323.060000.phy_data.tile6.nc \
+                   RESTART/20210323.060000.sfc_data.tile1.nc \
+                   RESTART/20210323.060000.sfc_data.tile2.nc \
+                   RESTART/20210323.060000.sfc_data.tile3.nc \
+                   RESTART/20210323.060000.sfc_data.tile4.nc \
+                   RESTART/20210323.060000.sfc_data.tile5.nc \
+                   RESTART/20210323.060000.sfc_data.tile6.nc \
+                   RESTART/20210323.060000.MOM.res.nc \
+                   RESTART/iced.2021-03-23-21600.nc \
+                   RESTART/ufs.cpld.cpl.r.2021-03-23-21600.nc \
+                   20210323.060000.out_pnt.ww3 \
+                   20210323.060000.out_grd.ww3 "
+
+export_fv3
+export_cpl
+
+export RESTART_N=12
+export RESTART_INTERVAL="${RESTART_N} -1"
+export OUTPUT_FH='0 24'
+
+export CPLCHM=.false.
+export DNATS=0
+export MOM6_TOPOEDITS=ufs.topo_edits_011818.nc
+export MOM6_ALLOW_LANDMASK_CHANGES=True
+export eps_imesh=2.5e-1
+
+export WW3_DOMAIN=global_270k
+export MESH_WAV=mesh.${WW3_DOMAIN}.nc
+export WW3_MODDEF=mod_def.ic4.exp.${WW3_DOMAIN}
+export WW3_ICE='C'
+export WW3_IC1='C'
+export WW3_IC5='C'
+
+export CICE_nfsd=12
+export CICE_tr_fsd=.true.
+export CICE_restart_fsd=.false.
+export CICE_wave_spec_type=constant
+
+export wav_coupling_to_cice=true
+export check_for_nans=true
+#export WW3_DTFLD=3600
+
+export DIAG_TABLE=diag_table_template
+export FIELD_TABLE=field_table_thompson_noaero_tke
+export UFS_CONFIGURE=ufs.configure.s2sw.IN
```

Comment:
This is not what we want to use. This config file places the waves in the slow loop. The original issue #1969 was to retain a fast-loop coupling test for the wave model by using the case where this fast-loop coupling is scientifically preferred (when coupled w/ the ice model in the fast loop). This test should use ``ufs.configure.s2sw_fast_esmf.IN``. 

The existing non-gfsv17 tests should be converted to use this wave-ice coupling configuration

https://github.com/ufs-community/ufs-weather-model/blob/4f32a4b5e6b17c85c6abf52ffd5b44a38d250c3b/tests/rt.conf#L77-L84

Reply:
Agreed. ufs.configure.s2sw.IN had waves in the fast loop back in December... Thanks @DeniseWorthen!

Reply:
Maybe I don't understand...you're saying all unstructured WW3 mesh tests should use -DPDLIB=ON -DWAVICE=ON?